valentyusb/usbcore/test/common.py

```diff
@@ -20,8 +22,29 @@ def grouper(n, iterable, pad=None):
     """
     return zip_longest(*[iter(iterable)]*n, fillvalue=pad)
 
-
-class CommonUsbTestCase(unittest.TestCase):
+# Test case helpers common to all test cases, simple and complex
+#
+class BaseUsbTestCase(unittest.TestCase):
+    # Create a name for the vcd file based on the test case module/class/method
```

Comment:
Convert these comments to docstrings....

Reply:
Converted.